tests/integration/component.go

```diff
@@ -308,8 +308,13 @@ func componentTests(args ...string) {
 			relativeContext := fmt.Sprintf("..%c%s", filepath.Separator, filepath.Base(commonVar.Context))
 			fmt.Printf("relativeContext = %#v\n", relativeContext)
 
-			helper.CmdShouldPass("odo", append(args, "create", "--s2i", "java:8", "sb-jar-test", "--project",
-				commonVar.Project, "--binary", filepath.Join(commonVar.Context, "sb.jar"), "--context", relativeContext)...)
+			if runtime.GOOS == "darwin" {
```

Comment:
whats the difference between these two conditions?

Reply:
```
helper.CmdShouldPass("odo", append(args, "create", "--s2i", "java:8", "sb-jar-test", "--project",
					commonVar.Project, "--binary", filepath.Join(commonVar.Context, "sb.jar"), "--context", relativeContext)...)
```

and 
```
helper.CmdShouldPass("odo", append(args, "create", "--s2i", "java:8", "sb-jar-test", "--project",
					commonVar.Project, "--binary", filepath.Join(commonVar.Context, "sb.jar"), "--context", relativeContext)...)
```

Reply:
For mac some how the test are using /private/var/.../afaf(random dir) to run test which is not the case for linux and windows, Changing the context passing according to working dir w.r.t test solves the issue.

Reply:
sorry but before there wasn't a "/private` in the filepath.Join, and I would suggest not force pushing the code

Reply:
Understood

Reply:
/approve